__docs__/wonder-blocks-theming/tokens-border.mdx

```diff
@@ -0,0 +1,96 @@
+import {Meta, Source} from "@storybook/blocks";
+
+import TokenTable from "../../.storybook/components/token-table";
+
+import Banner from "@khanacademy/wonder-blocks-banner";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+<Meta title="Theming / Tokens / Border" />
+
+export const baseColumns = (kind) => [
+    {
+        label: "Token",
+        cell: (row) => (
+            <code>
+                border.{kind}.{row.label}
+            </code>
+        ),
+    },
+    {
+        label: "Value",
+        cell: "value",
+    },
+];
+
+export const mapFunction = ([key, value]) => ({
+    label: key,
+    value: value,
+});
+
+export const tokensRadiusMap = Object.entries(tokens.border.radius).map(
+    mapFunction,
+);
+
+export const tokensWidthMap = Object.entries(tokens.border.width).map(
+    mapFunction,
+);
+
+# Tokens
+
+## Border
+
+<Banner
+    kind="warning"
+    text="Note that this is intended only internal
+    development purposes. We'll let you know as soon as this is ready for public
+    consumption."
```

Comment:
Layout is supposed to be required on banner 😅

Reply:
ahh hmmm that's probably one of the issues with MDX... it looks it doesn't enforce TS as we'd expect.